chromadb/utils/embedding_functions.py

```diff
@@ -4,14 +4,14 @@
 class SentenceTransformerEmbeddingFunction(EmbeddingFunction):
     # If you have a beefier machine, try "gtr-t5-large".
     # for a full list of options: https://huggingface.co/sentence-transformers, https://www.sbert.net/docs/pretrained_models.html
-    def __init__(self, model_name: str = "all-MiniLM-L6-v2"):
+    def __init__(self, model_name: str = "all-MiniLM-L6-v2", device: str = "cpu"):
```

Comment:
I was taking another look at this and I think we should swap the generic `str` for a python enum type that lists the available options. this avoids the user having to guess what strings are valid.

Reply:
Nevermind - going back on this. We don't want this enum.